scalding-beam/src/main/scala/com/twitter/scalding/beam_backend/BeamWriter.scala

```diff
@@ -48,7 +75,16 @@ class BeamWriter(val beamMode: BeamMode) extends Writer {
               }
               rec(xs)
             }
-            //TODO: handle Force and ToIterable
+            case OptimizedWrite(pipe, ToWrite.ToIterable(opt)) =>
+              val pcoll = planner(opt).run(pipeline)
```

Comment:
just to be clear: this isn't blocking right? this adds it to a pipeline which is run later below right?

Reply:
It only builds up on the pipeline object which runs in the end.